spiderable_server.js

```diff
@@ -16,6 +16,13 @@ var crypto = Npm.require('crypto');
 Spiderable.userAgentRegExps = [
     /^facebookexternalhit/i, /^linkedinbot/i, /^twitterbot/i];
```

Comment:
I am unsure about visionutils, or vkShare, but I don't see the harm.
Note remember to escape the . in the Mail.RU_Bot. (Mail\.RU_Bot)